source/application/models/oxarticle.php

```diff
@@ -555,6 +555,9 @@ public function getActiveCheckQuery($blForceCoreTable = null)
         // check if article is still active
         $sQ = " $sTable.oxactive = 1 ";
 
+        // check if article is hidden
```

Comment:
Iwould like to have an cosmetic change here, i think it's better document the "why" instead the "what" e.g.
//  hidden articles should not be included in lists e.g the category pages
Btw +1 for this pull request